overlord/devicestate/firstboot20_test.go

```diff
@@ -98,9 +98,17 @@ func (s *firstBoot20Suite) snapYaml(snp string) string {
 	return s.extraSnapYaml[snp]
 }
 
-func (s *firstBoot20Suite) setupCore20LikeSeed(c *C, sysLabel string, modelGrade asserts.ModelGrade, kernelAndGadget bool, extraGadgetYaml string, extraSnaps ...string) *asserts.Model {
+type core20SeedOptions struct {
+	sysLabel        string
+	modelGrade      asserts.ModelGrade
+	kernelAndGadget bool
+	extraGadgetYaml string
+	valsets         []string
+}
+
+func (s *firstBoot20Suite) setupCore20LikeSeed(c *C, opts *core20SeedOptions, extraSnaps ...string) *asserts.Model {
```

Comment:
core20SeedOptions maybe shouldn't be a pointer here

Reply:
Done!